### DISCUSSION TITLE
Is there a way to generate a partial part (filter by feature endpoints) of the openapi schema?

### DISCUSSION THREAD

**Discussion by linusang** [openapi-ts/openapi-typescript#1655] [General]
My organisation api schema is huge so I would like to be able to only generate some of the endpoints, for example `/feature-one/*` endpoints to a single Typescript definition. Also because we have different feature api endpoints development at the same time, it is always changing, so generate them by feature endpoints make sense and reduce our effort of merge conflicts in a giant Typescript definition file

**Comment by ghost** [comment]
I am looking at how to do this as well. Did you ever figure it out @linusang ?

Perhaps one of the Redocly configuration rules can help. If I find out, I'll post my answer here.

- https://redocly.com/docs/cli/decorators/filter-out/
- https://redocly.com/docs/cli/rules/path-excludes-patterns/

**Reply by ghost** [reply]
Looks like an issue has been created for this as well - #1682
